go/adbc/driver/databricks/driver_test.go

```diff
@@ -336,6 +355,12 @@ func TestDatabricks(t *testing.T) {
 	})
 }
 
+func TestDatabricksWithURI(t *testing.T) {
+	withQuirksURI(t, func(q *DatabricksQuirks) {
+		suite.Run(t, &DatabricksTests{Quirks: q})
+	})
+}
+
```

Comment:
I don't think we really need to run all tests again...just check that we can connect and execute a trivial query